docs/integrations/xgboost.rst

```diff
@@ -11,28 +13,50 @@ XGBoost is an optimized distributed gradient boosting library designed to be hig
    :target: ../_static/images/integrations/xgboost_0.png
    :alt: XGBoost overview
 
-
-The integration is implemented as an XGBoost callback and provides the following capabilities:
+The integration is implemented as XGBoost callback and provides the following capabilities:
 
 * |metrics| (train and eval) after each boosting iteration.
 * |model| (Booster) to Neptune after the last boosting iteration.
 * |feature| to Neptune as an image after the last boosting iteration.
 * |tree| to Neptune as images after the last boosting iteration.
 
-.. tip:: Try the integration right away with this |google-colab|.
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`Quickstart <quickstart-xgb>` below.
+
+If you want to try things out and focus only on the code you can either:
+
+#. Open Colab notebook (badge-link below) with quickstart code and run it as a "`neptuner`" user - zero setup, it just works,
+#. View quickstart code as a plain Python script on |script|.
+
+|Run on Colab|
+
+.. note::
+
+    This integration is tested with ``xgboost==1.2.0`` and current latest, and ``neptune-client==0.4.124`` and current latest.
```

Comment:
what does "and current latest" mean here -> got confused

Reply:
Dropped "current latest."